app/workers/fast_track/disability_compensation_job.rb

```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
+
+require 'prawn'
+require 'prawn/table'
+require 'lighthouse/veterans_health/client'
+
+module FastTrack
+  class DisabilityCompensationJob
+    include Sidekiq::Worker
+
+    extend SentryLogging
+    # NOTE: This is apparently at most about 4.5 hours.
+    # https://github.com/mperham/sidekiq/issues/2168#issuecomment-72079636
+    sidekiq_options retry: 8
+
+    sidekiq_retries_exhausted do |msg, _ex|
+      submission_id = msg['args'].first
+      submission = Form526Submission.new
+      submission.start_evss_submission(_status, submission_id: submission_id)
+    end
+
+    def perform(form526_submission_id, full_name)
```

Comment:
Nit: This method is quite long.

(You don't have to fix nits.)

Reply:
It is!! I look forward to refactoring this code further after we get this on staging. It's definitely doing a lot. I feel like that happens to workers a lot.